jerry-core/jmem/jmem-heap.c

```diff
@@ -615,12 +615,12 @@ jmem_heap_stats_print (void)
                    heap_stats->peak_object_bytes,
                    heap_stats->property_bytes,
                    heap_stats->peak_property_bytes,
-                   heap_stats->skip_count / heap_stats->nonskip_count,
-                   heap_stats->skip_count % heap_stats->nonskip_count * 10000 / heap_stats->nonskip_count,
-                   heap_stats->alloc_iter_count / heap_stats->alloc_count,
-                   heap_stats->alloc_iter_count % heap_stats->alloc_count * 10000 / heap_stats->alloc_count,
-                   heap_stats->free_iter_count / heap_stats->free_count,
-                   heap_stats->free_iter_count % heap_stats->free_count * 10000 / heap_stats->free_count);
+                   heap_stats->skip_count / (heap_stats->nonskip_count + 1),
+                   heap_stats->skip_count % (heap_stats->nonskip_count + 1) * 10000 / (heap_stats->nonskip_count + 1),
+                   heap_stats->alloc_iter_count / (heap_stats->alloc_count + 1),
+                   heap_stats->alloc_iter_count % (heap_stats->alloc_count + 1) * 10000 / (heap_stats->alloc_count + 1),
+                   heap_stats->free_iter_count / (heap_stats->free_count + 1),
+                   heap_stats->free_iter_count % (heap_stats->free_count + 1) * 10000 / (heap_stats->free_count + 1));
```

Comment:
because the unittest `test-poolmain,c` calls `jmem_heap_stats_print`, which will cause error if we don't fix it. (JMEM_STATS is not enabled before, so the issue is not be discovered)

What about comments out the function call, and I will reset the change for function body.